backend/api/activity/index.js

```diff
@@ -1,13 +1,35 @@
 const express = require('express');
+const Activity = require('../../database/models/activity');
 
 const activity = express.Router();
 
 /**
- * Returns activity data by id
+ * Creates a new activity and returns its data
+ * @body {String} title - The title of the post
+ * @body {String} description - The title of the post
+ */
+activity.route('/create')
+  .post(async(req, res) => {
+    const statusCode = 200;
+	const currentDate = Date.now();
+	const activity = new Activity({
+		title: req.body.title,
+		description: req.body.description,
+		creationDateTime: currentDate,
+		lastEditDateTime: currentDate,
+	})
+	await activity.save()
```

Comment:
Redefining the variable `activity` makes this code a little confusing to read. While redefining the variable in the scope of the callback should be semantically sound, it can make it harder to understand exactly what variable we're modifying and could potentially lead to problems. A different variable name would be preferable.

Also, down at the end of the function, when we're using `await` while saving a document we need to replace the callback that would normally catch any errors in saving the document. We can use a [`try...catch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) block to achieve this.
```js
try {
  await activity.save();
} catch(error) {
  // Handle saving errors and return an error response
}
```